org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/SonarLintUiPlugin.java

```diff
@@ -294,12 +294,13 @@ public IStatus run(IProgressMonitor monitor) {
           } else if (newestSonarLintVersion.isNewerThan(currentSonarLintVersion)) {
             NewerVersionAvailablePopup.displayPopupIfNotAlreadyShown(newestSonarLintVersion.toString());
           }
+
+          SonarLintGlobalConfiguration.setNextSonarLintVersionHintDate();
```

Comment:
Possibly I misunderstood, but shouldn't we reset the date only after the notification was shown? i.e. after line 295.